view/frontend/web/js/view/catalog/product/product-protection-modal-offer.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Extend (c) 2023. All rights reserved.
+ * See Extend-COPYING.txt for license details.
+ */
+
+define(['jquery', 'extendSdk', 'ExtendMagento'], function ($, Extend, ExtendMagento) {
+  'use strict'
+
+  return function (config, element) {
+    Extend.config({ storeId: config[0].extendStoreUuid, environment: config[0].activeEnvironment })
+
+    const productSku = config[0].productSku
+
+    const addToCartForm = $('#product_protection_modal_offer_' + productSku)
```

Comment:
I tried using vanilla js here but I couldn't find the initial div that way, not sure if jQuery waits for the document to be ready behind the scenes or something. If we care enough about avoiding using it I'm open to suggested vanilla js solutions.

Reply:
i think this is fine for now. my suggestion would be
```javascript
document.querySelector('#product_protection_modal_offer_' + productSku)
?.closest('.product.actions.product-item-actions')
?.querySelector('.actions-primary')
?.querySelector('form[data-role="tocart-form"]')
```
i wasn't aware of any differences between `.find()` and `.querySelector` in regards to timing with dom content loading. i think this is fine but we might want a tech debt ticket to replace this in the future if we can find a simple vanilla way to make it work.

Reply:
@jay-faust-extend I'd go with the alternative above suggested by Miles.

Reply:
@scott-taylor-extend I tried the above with no success, I'll try again